fvm/meter/basic/meter.go

```diff
@@ -17,21 +17,28 @@ type Meter struct {
 	computationLimit uint
 	memoryUsed       uint
 	memoryLimit      uint
+
+	computationIntensities map[uint]uint
+	memoryIntensities      map[uint]uint
```

Comment:
now that we have default weights we might not need these changes to the basic meter and return empty.